bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -303,6 +349,16 @@ spec:
       - name: run-ovn
         hostPath:
           path: /var/run/ovn
+      # Used for placement of ACL audit logs 
+      - name: node-log
+        hostPath: 
+          path: /var/log/ovn
+      - name: log-socket
```

Comment:
These should be the same - /dev/log and the journald /dev/log...

Reply:
for some reason I had to had both for the host's `journald` process to pickup the logs, but I will experiment some more and try to just have one socket here.